pyglossary/plugins/xdxf/__init__.py

```diff
@@ -125,6 +123,9 @@ class Reader:
 
 	_html: bool = True
 	_xsl: bool = False
+	_has_added_css: bool = False
```

Comment:
These class attrs are used for read/write options! Don't use them to keep an "instance state" that you then change internally.

Reply:
Switch to instance attrs.

Reply:
Done

Reply:
Also, fixed ruff errors